x-pack/plugins/security_solution/public/entity_analytics/pages/entity_store_management_page.tsx

```diff
@@ -300,6 +315,39 @@ export const EntityStoreManagementPage = () => {
         <FileUploadSection />
         <EuiFlexItem grow={2}>
           <EuiFlexGroup direction="column">
+            {initEntityEngineMutation.isError && (
+              <EuiCallOut
+                title={
+                  <FormattedMessage
+                    id="xpack.securitySolution.entityAnalytics.entityStoreManagementPage.errors.initErrorTitle"
+                    defaultMessage={'There was a problem initializing the entity store'}
+                  />
+                }
+                color="danger"
+                iconType="alert"
+              >
+                <p>
+                  {(initEntityEngineMutation.error as { body: { message: string } }).body.message}
+                </p>
+              </EuiCallOut>
+            )}
+            {deleteEntityEngineMutation.isError && (
+              <EuiCallOut
+                title={
+                  <FormattedMessage
+                    id="xpack.securitySolution.entityAnalytics.entityStoreManagementPage.errors.deleteErrorTitle"
+                    defaultMessage={'There was a problem deleting the entity store'}
+                  />
+                }
+                color="danger"
+                iconType="alert"
+              >
+                <p>
+                  {(deleteEntityEngineMutation.error as { body: { message: string } }).body.message}
```

Comment:
Is there a way we can add this to the mutation typing?

Reply:
probably, there's a refactor issue already [here](https://github.com/orgs/elastic/projects/668/views/55?pane=issue&itemId=85637908&issue=elastic%7Csecurity-team%7C11035)

I'll add a comment there